nautobot/dcim/models/device_components.py

```diff
@@ -179,12 +180,7 @@ def connected_endpoint(self):
 
 
 @extras_features(
-    "custom_fields",
-    "custom_validators",
-    "export_templates",
-    "graphql",
-    "relationships",
-    "webhooks",
+    "custom_fields", "custom_validators", "export_templates", "graphql", "relationships", "webhooks", "computed_fields"
```

Comment:
General nit: please alphabetize the `extras_features` items (`computed_fields` would thus be the first, not the last, in the listing), and please do keep them on separate lines if they previously were set up that way, as it's more readable and makes for clearer diffs.